### DISCUSSION TITLE
What are Mobility Groups?

### DISCUSSION THREAD

**Discussion by hathibelagal-dev** [InterDigitalInc/AdvantEDGE#229] [Q&A] [answered]
I'm sorry if this is a very naive question, but what are mobility groups? I understand that they can help in load balancing and transferring UE state from one MEC application instance to another, but I haven't seen any mention of them in the ETSI whitepapers yet.

Can you please tell me if mobility groups and the mobility group manager are concepts limited only to the AdvantEDGE platform? Are there any papers I can refer to in order to learn more about them?

Best,
Hathibelagal

**Comment by roymx** [comment, marked as answer]
Yes, these are not tied to ETSI.

They are a service we developed in the past for a drones use case similar to your V2X - and are not a standardized option.
